src/gmp.c

```diff
@@ -21104,49 +21305,7 @@ gmp_xml_handle_end_element (/* unused */ GMarkupParseContext* context,
       CLOSE (CLIENT_CREATE_REPORT_RR, ERRORS);
       case CLIENT_CREATE_REPORT_RR_ERRORS_ERROR:
         {
-          create_report_result_t *result;
-
-          assert (create_report_data->results);
-
-          if (create_report_data->result_scan_nvt_version == NULL)
-            create_report_data->result_scan_nvt_version = strdup ("");
-
-          if (create_report_data->result_severity == NULL)
```

Comment:
I'm not sure if this really matters in practice, but I think it would be safer if you still set `result_severity` and `result_threat` to "-3.0" and "Error" like before.